bin/lib/zopen-generate

```diff
@@ -172,6 +172,7 @@ touch "${name}port/.gitignore" && chtag -tc 819 "${name}port/.gitignore"
 cat <<EOT > "${name}port/.gitignore"
 log/
 install/
+${name}*/
```

Comment:
isn't this a fairly broad ignore entry? This could accidentally pull in directories people may develop, e.g. gittests (say) ... I am expecting this is for the particular tarball and git directories created from the extract?

Reply:
It could probably be improved to get the basename from the tar and git urls and add both using something similar to this code:
```
filename=$(basename "$url") # get basename
name="${filename%.*}" # strip out the extension
```

Reply:
I think the directory `gitpatches` will get caught too?

![gitport](https://github.com/ZOSOpenTools/meta/assets/234149/f45c7ac5-0fc0-4227-a181-d4eb819ecd08)

Reply:
@v1gnesh , is this PR ready or is there more to be done?

Reply:
We don't have a way to allow dirnames like `gitpatches`.
Per this PR, `gitport` will add `git*/` to `.gitignore`, and that'll add gitpatches/ to ignore as well.